app/views/miq_ae_tools/_resolve.html.haml

```diff
@@ -1,6 +1,8 @@
-- content_for :left do
-  = render :partial => "resolve_form"
-  = render :partial => "resolve_form_buttons"
-
 #main_div
```

Comment:
Should probably move `#main_div` to `resolve`, not `_resolve` .. because `_resolve` is included from `show` and `resolve`, and `show` already does have `#main_div`...